api/src/main/java/org/hawkular/apm/api/model/Constants.java

```diff
@@ -58,6 +58,32 @@ private Constants() {}
     public static final String COMPONENT_EJB = "EJB";
 
 
+    /**
+     * This constant represents the prefix used by all Hawkular APM state transferred between
+     * communication services.
+     */
+    public static final String HAWKULAR_APM_PREFIX = "HWKAPM";
+
+    /**
+     * This constant represents the interaction id exchanges between a sender and receiver.
+     */
+    public static final String HAWKULAR_APM_ID = HAWKULAR_APM_PREFIX + "ID";
```

Comment:
These are header fields passed on the wire, so wanted small but clearly scoped (i.e. hawkular apm poject) names.